src/freenet/node/FNPPacketMangler.java

```diff
@@ -150,6 +145,28 @@ public void run() {
 
         private long lastConnectivityStatusUpdate;
         private Status lastConnectivityStatus;
+        private AtomicInteger unmatchedCount = new AtomicInteger(0);
+        // FIXME make configurable.
+        static final int MAX_UNMATCHED_PACKETS_IN_INTERVAL = 10;
+        private int CLEAR_UNMATCHED_INTERVAL = (int) MINUTES.toMillis(5);
```

Comment:
This interval should be way smaller, and the limit way higher. I receive about 20 unmatched packets each minute on a moderately connected peer. We only need to throttle when we're under attack; presumably in such a way that our performance would severely suffer.

Receiving a up till some tens of unmatched packets each second is handled just fine (just a minor nuisance). Receiving 1000 unmatched packets per second for a few a second, followed by several minutes of no unmatched packets, still makes us largely unreachable for those seconds, possibly causing backoff.